tests/solana_tests/tags.rs

```diff
@@ -211,6 +211,40 @@ fn functions() {
     assert_eq!(func.tags[2].tag, "inheritdoc");
     assert_eq!(func.tags[2].value, "b");
     assert_eq!(func.tags[2].no, 0);
+
+    let ns = parse_and_resolve(
+        r#"
+        contract c is b {
+            /// @return x sadad
+            /// @param k is a boolean
+            /// @custom:smtchecker abstract-function-nondet
+            function foo(int x) public pure returns (int a, bool k) {}
+        }
+
+        contract b {}"#,
+        Target::Solana,
+    );
+
+    assert_eq!(ns.diagnostics.len(), 4);
+
+    assert_eq!(
+        ns.diagnostics.first_error(),
+        "tag '@return' no matching return value 'x'"
```

Comment:
This error sentence is a little unclear. How about writing `could not find return value 'x' to match with tag`?

Reply:
I've reworded it